sql/src/main/java/io/crate/action/sql/RowReceiverToResultReceiver.java

```diff
@@ -27,7 +27,7 @@
 
 import java.util.Set;
 
-class RowReceiverToResultReceiver implements RowReceiver {
+public class RowReceiverToResultReceiver implements RowReceiver {
```

Comment:
can this class be kept package local? Move it into `protocols/postgres` if necessary